mautic/mautic
# Fix dashboard report line graphs

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Fix line graph from report on ddashboard.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create report email stats and add email send graph
3. Try add this graph to dashboard
![image](https://user-images.githubusercontent.com/462477/123831089-3e1b3880-d904-11eb-9796-8907970bda40.png)

4. You see something like even If you expect real graph
![image](https://user-images.githubusercontent.com/462477/123831130-4a06fa80-d904-11eb-8acf-a9a9442c591f.png)

5. After PR should load properly
